Day 1/Let and Const.js

```diff
@@ -1,12 +1,12 @@
 function main() {
     // Write your code here. Read input using 'readLine()' and print output using 'console.log()'.
     const PI = Math.PI;
-    var r = parseInt(readLine());
+    let r = parseFloat(readLine());
     // Print the area of the circle:
-    var area = PI * r * r;
+    let area = PI * Math.pow(r,2);
     console.log(area);
     
     // Print the perimeter of the circle:
     var perimeter = 2 * PI * r;
     console.log(perimeter);
-    
\ No newline at end of file
+    

```

Comment:
Remove the whitespace here and I will merge this.

Reply:
Added the missing '}' to resolve your white space issue.
literally Braced this branch.

Reply:
moreover if that '+' sign you're referring . That's there for our commentary :)